src/odemis/acq/fastem.py

```diff
@@ -461,73 +484,20 @@ def pre_calibrate(self):
-                                               self._descanner,
-                                               self._detector,
-                                               self._detector.data,
-                                               self._ccd,
-                                               self._stage)
-
-            configure_hw.configure_asm(self._multibeam,
-                                       self._descanner,
-                                       self._detector,
-                                       self._detector.data,
-                                       asm_config,
-                                       upload=False)
-            # Image translation pre-alignment ensures that the image of the multiprobe is roughly centered on the
-            # mppc detector.
-            logging.debug("Running image translation pre alignment before ROA acquisition.")
-            descanner_offset = image_translation_pre_align.run_image_translation_pre_align(self._scanner,
-                                                                                           self._multibeam,
-                                                                                           self._descanner,
-                                                                                           self._detector,
-                                                                                           self._detector.data,
-                                                                                           self._ccd)
-
-            # Set the descanner offset to the value calibrated with the image translation pre-alignment.
-            asm_config["descanner"]["scanOffset"] = descanner_offset
-            logging.debug(f"Descanner offset set to: {descanner_offset}")
-            # Blank the beam to reduce beam damage on the sample.
-            self._scanner.blanker.value = True
-
-        except Exception as ex:
-            logging.warning("Exception during pre-calibration of the ROA acquisition.",
-                            exc_info=True)
-            raise
-        finally:
-            # Blank the beam after the acquisition is done.
-            self._scanner.blanker.value = True
-            if asm_config is None:
-                logging.warning("Failed to retrieve asm configuration, configure_asm cannot be executed.")
-            else:
-                # put system into state ready for next task
-                # upload=False: it is enough to set calibrated values on HW during following acquisition
-                configure_hw.configure_asm(self._multibeam,
-                                           self._descanner,
-                                           self._detector,
-                                           self._detector.data,
-                                           asm_config,
-                                           upload=False)
-
-            logging.debug("Finish pre-calibration.")
```

Comment:
I think some logging that we are in the pre-calibrations is nice to keep. I would leave this one, and add another `"starting pre-calibration"` at the beginning of this function.